source/json.c

```diff
@@ -104,6 +110,25 @@ int aws_json_value_add_to_object(
     return result;
 }
 
+int aws_json_value_add_to_object_c_str(struct aws_json_value *object, const char *key, struct aws_json_value *value) {
```

Comment:
trivial: we can make the old functions call new functions to avoid code duplication.

Reply:
good point. forgot about updating the old functions. done now.